Modules/Filtering/Smoothing/include/itkFFTDiscreteGaussianImageFilter.h

```diff
@@ -49,7 +50,7 @@ class FFTDiscreteGaussianImageFilterEnums
    */
   enum class KernelSource : uint8_t
   {
-    COMBINED_OPERATORS = 0,
+    OPERATORS = 0,
```

Comment:
Changes which are not related to increase of test coverage could be squashed into the previous commit.

Reply:
Is it preferable to try to keep separate commits or reasonable to simply squash all into one? I tried to keep things separated but with fixups the distinctions between operational/coverage fixes have become blurry.

Reply:
This is good as-is. While I would have preferred these few changes moved to the previous commit and left most of the changes in this commit as a separate commit, I assume you are already quite tired of this PR and it is fine to merge it if CI is green.

Reply:
> I assume you are already quite tired of this PR and it is fine to merge it if CI is green

😃 

Thanks @dzenanz , if CI fails I will look at refactoring commits along with any necessary fixes, otherwise I'll merge as-is.

Reply:
CI has passed, I will go ahead with the merge.